processor_test.go

```diff
@@ -26,35 +26,66 @@ func PanicStringContains(t *testing.T, s string) {
 	}
 }
 
+func createMockBuilder(t *testing.T) (*gomock.Controller, *builderMock) {
+	ctrl := NewMockController(t)
+	bm := newBuilderMock(ctrl)
+	bm.st = storage.NewMemory()
```

Comment:
I think this should be default in the buildermock, otherwise test users wil panic if they forget to add this step